### PR TITLE
reports: Add pareksha's GSoC report

### PR DESCRIPTION
**Pull request by pareksha**
https://deploy-preview-610--coala-projects.netlify.com/gsoc/2018/statusreport/pareksha

### CODE REVIEW

**Comment by jschwarzwalder** [comment]
I agree with @jayvdb 

Having a list of links under the next steps would be helpful for anyone who takes on this project next. 
Linking to the unresolved issues or PR where we had discussion would be useful.

File: reports/pareksha.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+   - details: >
+      language: Add `escape_quotes` to lang definition
+ - 22:
+   - repo: coala
+   - link: https://github.com/coala/coala/pull/5475
+   - details: >
+      Formatting: Add taste `min_file_length`
+ - 23:
+   - repo: cEPs
+   - link: https://github.com/coala/cEPs/pull/148
```
Comment (jayvdb) [inline reports/pareksha.md]: need a new PR for this, with only min_file_length in it

File: reports/pareksha.md [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+   - details: >
+      PEP8Bear: Migrate bear to aspects
+ - 21:
+   - repo: coala
+   - link: https://github.com/coala/coala/pull/5504
+   - details: >
+      language: Add `escape_quotes` to lang definition
+ - 22:
+   - repo: coala
+   - link: https://github.com/coala/coala/pull/5475
```
Comment (jayvdb) [inline reports/pareksha.md]: merged

File: reports/pareksha.md [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+   - details: >
+      PycodestyleBear: Migrate bear to aspects
+ - 20:
+   - repo: coala-bears
+   - link: https://github.com/coala/coala-bears/pull/2654
+   - details: >
+      PEP8Bear: Migrate bear to aspects
+ - 21:
+   - repo: coala
+   - link: https://github.com/coala/coala/pull/5504
```
Comment (jayvdb) [inline reports/pareksha.md]: needs an update

File: reports/pareksha.md [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+   - details: >
+      LineLengthBear: Migrate to aspects
+ - 19:
+   - repo: coala-bears
+   - link: https://github.com/coala/coala-bears/commit/c9e8df083c5cae5651b81c0efdc9da3d0f79cd0b
+   - details: >
+      PycodestyleBear: Migrate bear to aspects
+ - 20:
+   - repo: coala-bears
+   - link: https://github.com/coala/coala-bears/pull/2654
```
Comment (jayvdb) [inline reports/pareksha.md]: as the three below can all get merged, it is not worth mentioning an unmerged PR here.

describe it in the prose at the bottom, indicating that it is blocked on a bug in an upstream tool

**Comment by jayvdb** [comment]
another commit in https://gitlab.com/coala/coala-utils/graphs/master and 3 in https://gitlab.com/coala/pyprint/graphs/master

File: reports/pareksha.md [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+ - GitLab:
+   - username: pareksha
+   - link: https://gitlab.com/pareksha
+ - Gitter:
+   - username: pareksha
+   - link: https://gitter.im/pareksha
+email: pareksha.manchanda@gmail.com
+blog: https://pareksha.wordpress.com/
+activity:
+ - 0:
```
Comment (jayvdb) [inline reports/pareksha.md]: please do everything in https://github.com/coala/projects/issues/615 and #627

File: reports/pareksha.md [hunk 6]
```diff
@@ -0,0 +1,279 @@
...
+layout: report
+report_year: '2018'
+permalink: /GSoC/2018/StatusReport/pareksha
+student: Pareksha Manchanda
+organisation: coala
+organisation_link: https://coala.io
+project: Convert Bears To Aspects
+project_link: https://summerofcode.withgoogle.com/projects/#5476000192790528
+tarball: https://rawgit.com/pareksha/GSoC-2018-Report/master/pareksha_GSOC_tarball.tar.gz
+sha256sum: 8f5db13ba58d7088b0779970c9f99be882e2d2b90c46da966e36d16b85112253
```
Comment (jayvdb) [inline reports/pareksha.md]: there are 22 patches in the tarball, but 26 activities in the page

File: reports/pareksha.md [hunk 7]
```diff
@@ -0,0 +1,279 @@
...
+layout: report
+report_year: '2018'
+permalink: /GSoC/2018/StatusReport/pareksha
+student: Pareksha Manchanda
+organisation: coala
+organisation_link: https://coala.io
+project: Convert Bears To Aspects
+project_link: https://summerofcode.withgoogle.com/projects/#5476000192790528
+tarball: https://rawgit.com/pareksha/GSoC-2018-Report/master/pareksha_GSoC_tarball.tar.gz
+sha256sum: 415ab63a56bd14fbd1160bb0c662fbb7bf114bdbaf539fe6a1c29c71c19abd8c
```
Comment (jayvdb) [inline reports/pareksha.md]: this sha is incorrect

**Comment by jayvdb** [comment]
ack https://github.com/coala/projects/pull/610/commits/25487527d89e68611c8b9f1dd8d0abc15a8bd3f0

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**